tests/boards/nrf/dmm/src/main.c

```diff
@@ -522,6 +522,10 @@ static void stress_allocator(void *mem_reg, bool cached)
 	int rv;
 	uint32_t curr_use;
 
+	if (mem_reg == NULL) {
+		ztest_test_skip();
+	}
+
```

Comment:
To save CI a bit, we should probably not build and run this test suite if DMM is not enabled. Can testcase.yaml not be used to filter? or is there a reason to build and run this suite even though DMM is not enabled?

Reply:
It's used to confirm that dmm API turns into empty calls when SoC does not support it. I think it is worth keeping it because in some drivers DMM API is called unconditionally and test on that board ensures that it works.

Reply:
ok